packages/walletconnect-v2/src/index.spec.ts

```diff
@@ -19,38 +18,60 @@ const createTestEnvironment = (opts: Omit<WalletConnectOptions, 'projectId'>) =>
 const accounts = ['0x0000000000000000000000000000000000000000']
 const chains = [1, 2, 3]
 
+/*
+ * TODO: Move this to `@web3-react/types` and further narrow down types for `RequestArguments` 
```

Comment:
note: will be tackled as a next PR, so not opening up a dedicated todo in the JIRA.